packages/runtime-core/src/renderer.ts

```diff
@@ -621,15 +621,24 @@ function baseCreateRenderer(
         optimized,
       )
     } else {
-      patchElement(
-        n1,
-        n2,
-        parentComponent,
-        parentSuspense,
-        namespace,
-        slotScopeIds,
-        optimized,
-      )
+      if (n1.el && (n1.el as VueElement)._isVueCE) {
```

Comment:
Can be changed to `const isVueCE = !!(n1.el && (n1.el as any)._isVueCE)` `patchElement` may cause `n1` to be unmounted, but `n1.el` still has a value after `patchElement`

Reply:
Change applied